roles/os_hardening/tasks/minimize_access.yml

```diff
@@ -224,6 +229,7 @@
     owner: 'root'
     group: 'root'
     mode: '{{ os_mnt_var_log_audit_dir_mode }}'
+  when: var_log_audit_exists.stat.exists | bool
```

Comment:
Thanks! I'll go one step further and add another check. We have that variable that controls whether auditd should be installed and it fits here.

```suggestion
  when:
    - var_log_audit_exists.stat.exists | bool
    - os_auditd_enabled | bool
```

Reply:
I debated adding that as well. Feels right to me.